web/src/views/Submission.vue

```diff
@@ -0,0 +1,220 @@
+<template>
+  <v-container fluid>
+    <transition name="slide-y-transition" mode="out-in">
+      <div v-if="file" :key="file.id">
+        <div class="heading">
+          <h2 class="heading-title">
+            Submission by {{ file.extra.fullName ?? file.shortPath }}
+          </h2>
+          <div class="heading-subtitle text--secondary">
+            Relevant information about the current submission.
+          </div>
+        </div>
+
+        <v-row>
+          <v-col cols="12" md="8">
+            <v-card>
+              <v-card-title>Compare</v-card-title>
+              <v-card-subtitle class="pb-2">
+                Compare this submission with other submissions
+              </v-card-subtitle>
+
+              <submissions-pairs-table :file="file" />
+            </v-card>
+
+            <v-card class="mt-4">

```

Comment:
Should we hide the timeline if there is no timestamp data available?
If we would hide it, some users may be confused by it absence. So maybe we can add an information message on the homepage like "The dataset you analyzed did not contain timestamps, so some visualizations will not be available. Learn how to add metadata [here](https://dolos.ugent.be/guide/dodona.html)".

Reply:
I would not remove the card, as it would be confusing. I will change this to the message you provided.

Reply:
![image](https://user-images.githubusercontent.com/6013068/186377392-32dee4b8-31b6-4be3-97d5-0b4813dd6363.png)